commands/new_function.go

```diff
@@ -91,36 +87,19 @@ the "Dockerfile" lang type in your YAML file.
 		return fmt.Errorf("folder: %s already exists", functionName)
 	}
 
-	if err := os.Mkdir("./"+functionName, 0700); err == nil {
+	if err := os.Mkdir(functionName, 0700); err == nil {
```

Comment:
Are you sure that this is exactly the same behaviour as previously? Why did it change?

Reply:
I changed it to avoid using concatenation in file path operations as suggested in another PR. The behaviour is the same.

"./" is no need, the path is relative to current working dir